### PR TITLE
Release DKP-Insights v0.1.5

### PR DESCRIPTION
**Pull request by asekretenko**

### CODE REVIEW

**Comment by asekretenko** [comment]
Upgrade tested on top of Kommander running v0.1.4 by re-pointing the GitRepository to this branch and bumping the version in the AppDeployment.

File: hack/images.yaml [hunk 1]
```diff
@@ -13,3 +13,8 @@ dkp-insights-0.1.4:
  - "docker.io/bitnami/bitnami-shell:10-debian-10-r305"
  - "docker.io/bitnami/postgres-exporter:0.10.0-debian-10-r172"
  - "docker.io/mesosphere/insights:v0.1.4"
+dkp-insights-0.1.4:
```
Comment (kaiwalyajoshi) [inline hack/images.yaml:16]:
```suggestion
dkp-insights-0.1.5:
```

File: services/dkp-insights/0.1.5/defaults/cm.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      repository: "mesosphere/insights"
+      tag: "v0.1.5"
+      imagePullPolicy: "IfNotPresent"
+    backend:
+       log_level: "INFO"
+    cleanup:
+      schedule: "@every 37m"
+      logLevel: "DEBUG"
+      insightsTTL: "168h"
+      eventsTTL: "168h"
```
Comment (kaiwalyajoshi) [inline services/dkp-insights/0.1.5/defaults/cm.yaml:20]: Did we want to change anything in `eventsTTL` to match: https://github.com/mesosphere/dkp-insights/pull/109/files

**Review by kaiwalyajoshi — CHANGES_REQUESTED** [review]

File: services/dkp-insights/0.1.5/defaults/cm.yaml (hunk 2 above)
Comment (asekretenko) [reply services/dkp-insights/0.1.5/defaults/cm.yaml:20]: No, we don't want to change the default here.

Btw, looks like I made a typo in #109 (there intention was not only to remove boundary from `--insights-ttl` but to use `max` for --events-ttl). Not critical for this release, though.

**Pull request merged**